src/compiler/data.ts

```diff
@@ -410,6 +415,37 @@ export namespace stack {
   };
 }
 
+export namespace dates {
+  /**
+   * Add data source for with dates for all months, days, hours, ... as needed.
+   */
+  export function defs(model: Model) {
+    let defs = [];
```

Comment:
I know that but if you push to the array, the reference to the array doesn't change.   Doesn't matter anymore anyway.